Content/Items/Forest/Armors.SlimePrince.cs

```diff
@@ -0,0 +1,273 @@
+﻿using StarlightRiver.Core.Systems.ExposureSystem;
+using Terraria.Audio;
+using Terraria.GameContent.ItemDropRules;
+using Terraria.ID;
+using static Terraria.ModLoader.ModContent;
+
+namespace StarlightRiver.Content.Items.Forest
+{
+	[AutoloadEquip(EquipType.Head)]
+	public class SlimePrinceHead : ModItem
+	{
+		public Projectile prince;
+		public Vector2 targetVel;
+		public float accel = 0.06f;
+		public Vector2 targetAccel;
+
+		public SlimePrinceMinion Minion => prince?.ModProjectile as SlimePrinceMinion;
+
+		public override string Texture => AssetDirectory.ForestItem + Name;
+
+		public override void Load()
+		{
+			StarlightPlayer.PreUpdateMovementEvent += SlimeMovement;
+			On_Player.KeyDoubleTap += ActivateMerge;
+
+			StarlightPlayer.FreeDodgeEvent += AbosrbSlime;
+		}
+
+		public override void SetStaticDefaults()
+		{
+			DisplayName.SetDefault("Slime Prince's Crown");
+			Tooltip.SetDefault("10% increased summoning damage");
+
+			ArmorIDs.Head.Sets.DrawFullHair[Item.headSlot] = true;
+		}
+
+		public override void SetDefaults()
+		{
+			Item.width = 18;
+			Item.height = 18;
+			Item.value = 1;
+			Item.rare = ItemRarityID.Green;
+			Item.defense = 2;
+		}
+
+		public override void UpdateArmorSet(Player player)
+		{
+			player.setBonus = "A slime prince follows you around\nDouble tap DOWN to fuse with the prince\nYou can control the prince during this time\nThe prince takes damage instead of you during this time";
+
+			// If the prince is invalid, we need to spawn a new prince
+			if (prince is null || !prince.active || prince.type != ProjectileType<SlimePrinceMinion>() || prince.owner != player.whoAmI)
+				prince = Projectile.NewProjectileDirect(player.GetSource_FromThis(), player.Center, Vector2.Zero, ProjectileType<SlimePrinceMinion>(), 28, 0, player.whoAmI);
```

Comment:
gonna start calling out more multiplayer compatibility stuff:
the prince variable will only get assigned to by the wearer of the helm so everything that tries to access 
SlimePrinceHead.Minion needs to be functionally owner only or this needs to get sent as a packet or searched for by the other clients and server